nodes/vr_vjunosevolved/vr-vjunosevolved.go

```diff
@@ -17,7 +17,7 @@ import (
 )
 
 var (
-	kindnames          = []string{"juniper_vjunosevolved"}
```

Comment:
we also are getting away from `vr-` named kinds, so keep only `juniper_vjunos...` in the kind names